schemas/yazi.json

```diff
@@ -27,14 +27,25 @@
 						"created",
 						"modified",
 						"natural",
-						"size"
+						"size",
+						"extention",
```

Comment:
Maybe we should sort them by their first letters here?

Reply:
order of doc seems better for doing quick comparisons between them